### ISSUE TITLE
Replace ftp with fragment update

### ISSUE THREAD

**Issue by maartends** [viaacode/vrt_avo_transfer#10]
Metadata update is pushed via FTP PUT as `<pid>.xml` to tra-server:

https://github.com/viaacode/vrt_avo_transfer/blob/5f90484f297d96484373ec1b832d6a74a63dcab7/src/main/app/poller.xml#L53

Replace with metadata-update via POST on `fragment_id`.

Needed: translation from `media_id` (VRT) to `fragment_id`.

**Comment by maartends** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[vrt_avo_transfer] Issue met metadata user op ftp fixen](https://trello.com/c/TB87vmRe/581-vrtavotransfer-issue-met-metadata-user-op-ftp-fixen)

**Comment by maartends** [comment]
Transfer of XML-metadata file could be simply replaced with a call/post to viaacode/mam-update-service.

**Comment by dietervanhoof** [comment]
Translation from pid -> fragment ID seems more logical to me. Media ID to fragment ID is harder.

**Comment by maartends** [comment]
Agreed. However, is it at all possible without an intermediate step?

`media_id -> pid -> frag_id` ?

**Comment by dietervanhoof** [comment]
Possibly with a custom query to MH based on the local cp ID.
